crates/adapters/dydx/src/python/websocket.rs

```diff
@@ -91,18 +252,13 @@ impl DydxWebSocketClient {
         py: Python<'py>,
         instrument_id: InstrumentId,
     ) -> PyResult<Bound<'py, PyAny>> {
-        let handler_cmd_tx = self.handler_cmd_tx.clone();
+        let client = self.clone();
         pyo3_async_runtimes::tokio::future_into_py(py, async move {
-            let ticker = ticker_from_instrument_id(&instrument_id);
-            let sub = crate::websocket::messages::DydxSubscription {
-                op: crate::websocket::enums::DydxWsOperation::Unsubscribe,
-                channel: crate::websocket::enums::DydxWsChannel::Trades,
-                id: Some(ticker),
-            };
-            let payload = serde_json::to_string(&sub).map_err(to_pyvalue_err)?;
-            handler_cmd_tx
-                .send(crate::websocket::handler::HandlerCommand::SendText(payload))
-                .map_err(|e| to_pyvalue_err(anyhow::anyhow!("{}", e)))
+            client
+                .unsubscribe_trades(instrument_id)
+                .await
+                .map_err(to_pyvalue_err_dydx)?;
```

Comment:
👌 `to_pyvalue_err_dydx` is good, as long as it's defined in the `nautilus-dydx` crate.